src/alr/alr-commands-show.ads

```diff
@@ -21,7 +21,7 @@ package Alr.Commands.Show is
       Config : in out CLIC.Subcommand.Switches_Configuration);
 
    overriding function Short_Description (Cmd : Command) return String is
-     ("See information about a release");
+     ("Display information about a crate");
```

Comment:
There's actually a somewhat of a fine distintion between crate and release. A release is a version of a crate, and the show command actually operates on a concrete version (the last one by default). Perhaps `crate version` is better if the idea is to avoid `release` as not to confuse users unnecessarily.